code/modules/client/preference_setup/loadout/items/augments.dm

```diff
@@ -299,17 +299,93 @@
+	path = /obj/item/organ/internal/augment/bioaug/auxiliary_heart
+	origin_restriction = list(/singleton/origin_item/origin/galatea)
+	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+	cost = 2
+
+/datum/gear/augment/boosted_heart
+	display_name = "Boosted Heart"
+	description = "Intended for athletes, some workers, and soldiers, this improved heart increases blood flow and circulation. It provides an improvement to blood oxygenation and stamina, at the cost of requiring more food and water. Outside of Galatea, this augment is popular among professional athletes."
+	path = /obj/item/organ/internal/heart/boosted_heart
+	origin_restriction = list(/singleton/origin_item/origin/galatea)
+	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+	cost = 3
+
+/datum/gear/augment/boosted_liver
+	display_name = "Boosted Liver"
+	description = "Designed primarily for diplomats or Galateans abroad, the boosted liver improves toxin filtering, giving a resistance to toxin damage. As a consequence, it makes it impossible for the user to get drunk."
+	path = /obj/item/organ/internal/liver/boosted_liver
+	origin_restriction = list(/singleton/origin_item/origin/galatea)
+	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+	cost = 2
+
 /datum/gear/augment/mind_blanker
 	display_name = "Galatean Mind Blanker"
 	description = "A small, discrete organ attached near the base of the brainstem. Any attempt to read the mind of an individual with this augment installed will fail, as will attempts at psychic brainwashing."
-	path = /obj/item/organ/internal/augment/mind_blanker
+	path = /obj/item/organ/internal/augment/bioaug/mind_blanker
 	origin_restriction = list(/singleton/origin_item/origin/galatea)
-	cost = 4
+	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+	cost = 2
 
 /datum/gear/augment/mind_blanker_lethal
 	display_name = "Galatean Mind Blanker (Lethal)"
 	description = "Available only to higher-up MfAS agents and members of the Galatean government. This enhanced variant of a mind blanker introduces a psionic trap which inflicts severe neural damage on anyone attempting to read the user's mind."
-	path = /obj/item/organ/internal/augment/mind_blanker_lethal
+	path = /obj/item/organ/internal/augment/bioaug/mind_blanker_lethal
+	origin_restriction = list(/singleton/origin_item/origin/galatea)
+	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD)
+	cost = 4
+	allowed_roles = list("Consular Officer")
+
+/datum/gear/augment/platelet_factories
+	display_name = "Platelet Factories"
+	description = "Designed for military applications, this implant massively increases the user's blood clotting factor. This provides an extreme resistance to arterial bleeds, effectively all but preventing exsanguination. This augment has a reputation for causing heart attacks and strokes at a high rate, and is usually combined with an auxiliary heart for safety."
```

Comment:
After a conversation with the loremasters, explicitly military bioaugmentations will not be allowed in the loadout.